nixos/tests/typesense.nix

```diff
@@ -18,6 +18,9 @@ in {
   testScript = ''
     machine.wait_for_unit("typesense.service")
     machine.wait_for_open_port(${toString testPort})
+    # Just waiting for the port is too fast for typesense to initialize the database
+    # Wait a little more.
+    machine.sleep(5)
```

Comment:
I think either the service itself should support systemd notification, or we can use `wait_until_succeeds`, sleeping for a while is still unreliable on slow machines.

Reply:
Good point, that's better.